src/efs/efs3.py

```diff
@@ -0,0 +1,58 @@
+"""Eatfirst FileSystem for S3."""
+
+from autorepr import autorepr
+from fs.path import iteratepath
+from fs.path import normpath
+from fs.path import relpath
+from fs.s3fs import S3FS
+from fs.s3fs import thread_local
+
+
+class EFS3(S3FS):
```

Comment:
Now that we are messing with this untouched part of the code... why not `EatFirstS3`?